eng/native/build-commons.sh

```diff
@@ -79,15 +79,21 @@ build_native()
     fi
 
     if [[ "$targetOS" == android || "$targetOS" == linux-bionic ]]; then
+        # Keep in sync with $(AndroidApiLevelMin) in Directory.Build.props in the repository rooot
+        local ANDROID_API_LEVEL=21
         if [[ -z "$ANDROID_NDK_ROOT" ]]; then
             echo "Error: You need to set the ANDROID_NDK_ROOT environment variable pointing to the Android NDK root."
             exit 1
         fi
 
+        # cmake cache scripts can't see command line args
+        ANDROID_BUILD=1
+        export ANDROID_BUILD
+
         cmakeArgs="-C $__RepoRootDir/eng/native/tryrun.cmake $cmakeArgs"
 
         # keep ANDROID_PLATFORM in sync with SetOSTargetMinVersions in the root Directory.Build.props
```

Comment:
Nit: Is this comment still needed? (Looks like it essentially moved a couple lines above.)